README.md

```diff
@@ -14,11 +14,27 @@ npm install -g webpack-cli
 **NOTE: FOR RUNNING AND DEVELOPING YOU NEED TO ACTIVATE YOUR Red Hat VPN.**
 
 ## Cloning the repositories
+You'll need to clone these repositories:
 
-* You can curl a script to bypass the downloading. You will still have to run each application yourself as noted in the "Running and Developing" section below *
+``` bash
+git clone https://github.com/project-xavier/xavier-ui.git
+cd xavier-ui
+yarn install
+
+git clone https://github.com/carlosthe19916/insights-chrome.git
+cd insights-chrome
+yarn install
+
+git clone https://github.com/RedHatInsights/insights-proxy.git
+cd insights-proxy
+yarn install
+```
 
+(Optional) You can also clone jaxrs-util-mocks for having jax-rs mocks
```

Comment:
Maybe better to also add the reason to clone this repo.
Something like:
```suggestion
(Optional) You can also clone jaxrs-util-mocks for having jax-rs mocks in order to be able to develop locally without any need of having a running backend
```
but feel free to describe it as you prefer.